user-manager/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 plugins {
     id("org.springframework.boot")
     id("io.spring.dependency-management")
```

Comment:
We don't need these spring plugins anymore here and in all other modules where you use your custom one